app/controllers/venues/managers_controller.rb

```diff
@@ -6,7 +6,7 @@ def create
   end
 
   def destroy
-    venue = current_user.venues.find_by(url:params[:venue])
+    venue = find_venue
```

Comment:
Don't we want to leave this as current_user to we can't open a security gap? Or are there checks prior?